### DISCUSSION TITLE
How to not compress the source code？

### DISCUSSION THREAD

**Discussion by hollownicky** [vercel/next.js#35425] [Help]
**Now:** 
![image](https://user-images.githubusercontent.com/70936757/158956810-540d6da4-cc4b-428b-af41-3f7a757c5ed1.png)
![image](https://user-images.githubusercontent.com/70936757/158956830-c13d1cb9-7388-4832-b84c-2c5a2021988a.png)

**I Need:**
![image](https://user-images.githubusercontent.com/70936757/158956866-97b9c702-6047-4d7f-b2ab-6dd38c92816d.png)
![image](https://user-images.githubusercontent.com/70936757/158955602-3312fd47-9334-434b-990c-3a21baa0df10.png)

I used the following method, but doesn't work.
`const nextConfig = {
     compress: false
}
module.exports = nextConfig`

**Comment by icyJoseph** [comment]
https://github.com/vercel/next.js/discussions/21293 it is **not possible.** 

For **JavaScript** you can do:

```
// next.config.js 
/** @type {import('next').NextConfig} */
const config = {
  reactStrictMode: true,
  webpack(config) {
    config.optimization.minimize = false;
    config.optimization.minimizer = [];

    return config;
  },
};

module.exports = config;
```

It is possible to use tools to `beautify` html.
